docs/01_nodeos/03_plugins/http_client_plugin/index.md

```diff
@@ -1,39 +0,0 @@
-## Description
```

Comment:
Why was this file completely removed?

Reply:
After I removed the ssl specific stuff, the file was almost empty, and it was suggested that I remove the file altogether.

Reply:
Sorry I missed Matt and Ovi's comments about that.